tests/scripts/start-storybook.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+npx kill-port 6006
 npm run storybook -- --quiet --ci &
```

Comment:
is there a start-storybook cli flag we can use to enforce the port number and fail if the port is already in use? for instance vite has https://vitejs.dev/config/server-options.html#server-strictport . 
I believe this is the storybook cli docs? https://storybook.js.org/docs/react/api/cli-options

Reply:
Do we want the start-storybook to fail if the port is in use or kill whatever process that's running to start the test? I don't think there is a cli option in the storybook docs that does this.

Reply:
hmm I think having storybook fail instead of kill whatever was already on 6006 might be better? killing a preexisting process seems a little aggressive

Reply:
Okay I updated the test script to make the test fail and asking the user to kill process instead of killing the process automatically